payments-core/src/main/java/com/stripe/android/payments/core/analytics/ErrorReporter.kt

```diff
@@ -50,6 +50,25 @@ interface ErrorReporter {
                 .build()
                 .errorReporter
         }
+
+        fun getAdditionalParamsFromError(error: Throwable): Map<String, String> {
+            return when (error) {
+                is StripeException -> getAdditionalParamsFromStripeException(error)
+                else -> getAdditionalParamsFromStripeException(StripeException.create(error))
+            }
+        }
+        fun getAdditionalParamsFromStripeException(stripeException: StripeException): Map<String, String> {
+            return mapOf(
+                "analytics_value" to stripeException.analyticsValue(),
+                "status_code" to stripeException.statusCode.toString(),
```

Comment:
```suggestion
                "status_code" to stripeException.statusCode?.toString(),
```

Reply:
as discussed in slack, it's sending a 0 but I'll send out a follow up PR to make statusCode nullable and then add the ? back